src/nav/apm.yml

```diff
@@ -1310,10 +1310,8 @@ pages:
             path: /docs/apm/transactions/transaction-traces/introduction-transaction-traces
           - title: Configure transaction traces
             path: /docs/apm/transactions/transaction-traces/configure-transaction-traces
-          - title: Summary page
-            path: /docs/apm/transactions/transaction-traces/transaction-traces-summary-page
           - title: Trace details page
-            path: /docs/apm/transactions/transaction-traces/transaction-traces-trace-details-page
+            path: /docs/apm/transactions/transaction-traces/transaction-trace-details-page
```

Comment:
@homelessbirds this gives me a 404, page not found error.

I was able to get this URL to work but it's really the same. Still, there is no sign of it in the right navigation pane:

[/docs/apm/transactions/transaction-traces/transaction-traces-trace-details-page/](https://docswebsitedevelop-sdtransactiontracev1.gatsbyjs.io/docs/apm/transactions/transaction-traces/transaction-traces-trace-details-page/)

Reply:
That's what I had originally. Reverting it back. Thanks!

Reply:
Done.